Keas.Mvc/Program.cs

```diff
@@ -89,7 +89,7 @@ static LoggerConfiguration CreateLogConfig(IConfigurationRoot configuration)
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
                 // .MinimumLevel.Override("Microsoft.EntityFrameworkCore", LogEventLevel.Warning) // uncomment this to hide EF core general info logs
                 .MinimumLevel.Override("Microsoft.AspNetCore", LogEventLevel.Warning)
-                .Enrich.WithElasticApmCorrelationInfo()
+                //.Enrich.WithElasticApmCorrelationInfo()

```

Comment:
Does it add much to the logs if we leave the correlation? I think if we remove our stackify logger we would want to keep this apm enriched.